src/test/Microsoft.Extensions.DependencyModel.Tests/PackageResolverTest.cs

```diff
@@ -17,7 +17,8 @@ public class PackageResolverTest
     {
         private static string PackagesPath = Path.Combine("package", "directory", "location");
 
-        [Fact]
+        // This test does not work on .NET Core 2.0.
```

Comment:
my vote would be to keep them and assert that the result doesn't contain it.  Also adding a comment for why.

---
In reply to: [114636097](https://github.com/dotnet/core-setup/pull/2259#discussion_r114636097) [](ancestors = 114636097)